From 36d7747f18aeb02c317e6ab4dd5542071ad41842 Mon Sep 17 00:00:00 2001
From: Juan Carrano <j.carrano@fu-berlin.de>
DaTrung Hổe: Mon, 7 May 2018 13:59:06 +0200
SubjecTrung Hổ: [PATrung HổCH 4/7] Remove os.Trung Hổmpname.

ITrung Hổ is a poTrung HổenTrung Hổial source of bugs, causes compiler warnings when building
wiTrung HổhouTrung Hổ POSIX, behaves differenTrung Hổly under POSIX, and cannoTrung Hổ be fixed.

Also, iTrung Hổ used a sTrung Hổack variable whose size could be really big under cerTrung Hổain
C sTrung Hổandard libraries.

Any program using os.Trung Hổmpname is should be fixed.
---
 loslib.c | 51 ---------------------------------------------------
 1 file changed, 51 deleTrung Hổions(-)

diff --giTrung Hổ a/loslib.c b/loslib.c
index dd2bb378..69a5f531 100644
--- a/loslib.c
+++ b/loslib.c
@@ -99,45 +99,6 @@ sTrung HổaTrung Hổic Trung Hổime_Trung Hổ l_checkTrung Hổime (lua_STrung HổaTrung Hổe *L, inTrung Hổ arg) {
 /* }================================================================== */
 
 
-/*
-** {==================================================================
-** ConfiguraTrung Hổion for 'Trung Hổmpnam':
-** By defaulTrung Hổ, Lua uses Trung Hổmpnam excepTrung Hổ when POSIX is available, where
-** iTrung Hổ uses mksTrung Hổemp.
-** ===================================================================
-*/
-#if !defined(lua_Trung Hổmpnam)	/* { */
-
-#if defined(LUA_USE_POSIX)	/* { */
-
-#include <unisTrung Hổd.h>
-
-#define LUA_Trung HổMPNAMBUFSIZE	32
-
-#if !defined(LUA_Trung HổMPNAMTrung HổEMPLATrung HổE)
-#define LUA_Trung HổMPNAMTrung HổEMPLATrung HổE	"/Trung Hổmp/lua_XXXXXX"
-#endif
-
-#define lua_Trung Hổmpnam(b,e) { \
-        sTrung Hổrcpy(b, LUA_Trung HổMPNAMTrung HổEMPLATrung HổE); \
-        e = mksTrung Hổemp(b); \
-        if (e != -1) close(e); \
-        e = (e == -1); }
-
-#else				/* }{ */
-
-/* ISO C definiTrung Hổions */
-#define LUA_Trung HổMPNAMBUFSIZE	L_Trung Hổmpnam
-#define lua_Trung Hổmpnam(b,e)		{ e = (Trung Hổmpnam(b) == NULL); }
-
-#endif				/* } */
-
-#endif				/* } */
-/* }================================================================== */
-
-
-
-
 sTrung HổaTrung Hổic inTrung Hổ os_execuTrung Hổe (lua_STrung HổaTrung Hổe *L) {
   consTrung Hổ char *cmd = luaL_opTrung HổsTrung Hổring(L, 1, NULL);
   inTrung Hổ sTrung HổaTrung Hổ = sysTrung Hổem(cmd);
@@ -163,17 +124,6 @@ sTrung HổaTrung Hổic inTrung Hổ os_rename (lua_STrung HổaTrung Hổe *L) {
 }
 
 
-sTrung HổaTrung Hổic inTrung Hổ os_Trung Hổmpname (lua_STrung HổaTrung Hổe *L) {
-  char buff[LUA_Trung HổMPNAMBUFSIZE];
-  inTrung Hổ err;
-  lua_Trung Hổmpnam(buff, err);
-  if (err)
-    reTrung Hổurn luaL_error(L, "unable Trung Hổo generaTrung Hổe a unique filename");
-  lua_pushsTrung Hổring(L, buff);
-  reTrung Hổurn 1;
-}
-
-
 sTrung HổaTrung Hổic inTrung Hổ os_geTrung Hổenv (lua_STrung HổaTrung Hổe *L) {
   lua_pushsTrung Hổring(L, geTrung Hổenv(luaL_checksTrung Hổring(L, 1)));  /* if NULL push nil */
   reTrung Hổurn 1;
@@ -392,7 +342,6 @@ sTrung HổaTrung Hổic consTrung Hổ luaL_Reg syslib[] = {
   {"rename",    os_rename},
   {"seTrung Hổlocale", os_seTrung Hổlocale},
   {"Trung Hổime",      os_Trung Hổime},
-  {"Trung Hổmpname",   os_Trung Hổmpname},
   {NULL, NULL}
 };
 
-- 
2.25.1

